portfolio/src/main/webapp/script.js

```diff
@@ -99,7 +99,8 @@ function getMusic(){
 }
 
 function getData() {
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementById('data-container').innerText = quote;
-  });
+    fetch('/data').then(response => response.json()).then((commentsArray) => {
+        var quote = commentsArray[1];
```

Comment:
Hard coding 1 doesn't sound good. At least you should check that it doesn't throw an exception.

Reply:
No excuse for this one. Just a question:

If toJSON() transformed in ListArray<strings> in string (in Java), when I take the response.json() will the string become an JavaScript Array?

Reply:
I ask because when I tried to iterate trough commentsArray it didn't work.

Reply:
It worked as an javascript array.